### ISSUE TITLE
Two-way SMS supports in uplink gateway

### ISSUE THREAD

**Issue by antonraharja** [antonraharja/playSMS#114]
From Faizan Mirza:

Hello, I am using kannel with playsms on local system with GSM Devices connected. SMS going and coming to admin panel. I have uploaded also playsms to my hosting with uplink gateway. Messages going but how to get messages in inbox to make hosted playsms as two way sms system? I have low bandwidth so cant use local playsms for worldwide access. Please tell me what are possibilities and what sms sync help us..

Basically a way to make uplink gateway become two-way instead of one way like now.

**Comment by antonraharja** [comment]
dropped in favor of Playnet
